pallets/hyperdrive-outgoing/src/types.rs

```diff
@@ -0,0 +1,272 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core::fmt::Debug;
+pub use mmr_lib;
+
+use frame_support::pallet_prelude::*;
+use scale_info::TypeInfo;
+use sp_core::RuntimeDebug;
+use sp_runtime::traits;
+#[cfg(not(feature = "std"))]
+use sp_std::prelude::Vec;
+use sp_std::prelude::*;
+
+use pallet_acurast::{JobIdSequence, TezosAddressBytes};
+use strum_macros::{EnumString, IntoStaticStr};
+
+/// Prefix for elements stored in the Off-chain DB via Indexing API.
+pub const INDEXING_PREFIX: &'static [u8] = b"ammr";
```

Comment:
The prefix should probably contain the target chain identifier.

Reply:
that was a left-over, the prefix is specified in the config, specific for each instance=target chain